.gitignore

```diff
@@ -1,3 +1,5 @@
 dist/
 node_modules/
-target/
\ No newline at end of file
+target/
+examples/signature-agent-card-and-registry/build/
+examples/signature-agent-card-and-registry/.wrangler/
```

Comment:
you can add a `.gitignore` directly in `examples/signature-agent-card-and-registry`

if `build` and `.wrangler` are meant to be ignored in all directory, you can add them directly without the full path

Reply:
done